### PR TITLE
Changed readme file descriptions.

### PR DESCRIPTION
**Pull request by LavanyaAnaimuthu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 * Pyramid chart
 * Funnel chart
-* Polar chart
+* Polar charts
 
-#### Cartesian chart
+#### Cartesian charts
 
-A Cartesian chart is an elegant way to visualize data. Charts supports more than 15 Cartesian chart types such as line, spline, column, bar, area, scatter, and bubble.
+A Cartesian chart is an elegant way to visualize data. Charts supports more than 15 Cartesian chart types such as line, spline, column, area, scatter, and bubble.
```
Comment (E-Mani) [inline README.md]: The Cartesian chart is an elegant way to visualize data. It supports more than 15 Cartesian chart types such as line, spline, column, area, scatter, bubble, etc.

File: README.md [hunk 2]
```diff
@@ -2,23 +2,23 @@
 
 This repository contains the samples that demonstrate the functionalities of WinUI chart controls.
```
Comment (E-Mani) [inline README.md:4]: WinUI Chart controls.

File: README.md [hunk 3]
```diff
@@ -2,23 +2,23 @@
...
+A Cartesian chart is an elegant way to visualize data. Charts supports more than 15 Cartesian chart types such as line, spline, column, area, scatter, and bubble.
 
 [CartesianChart Sample](https://github.com/SyncfusionExamples/GettingStartedChartWinUI/tree/main/CartesianChartGettingStarted)
 
 [CartesianChart UG](https://help.syncfusion.com/winui/cartesian-charts/getting-started)
 
-#### Circular chart
+#### Circular charts
 
 A circular chart is user-friendly and gives greater UI visualization of data with proportions and percentages of different categories.
```
Comment (E-Mani) [inline README.md]: The Circular chart is

File: README.md (hunk 2 above)
Comment (LavanyaAnaimuthu) [reply README.md:4]: changed.

File: README.md (hunk 1 above)
Comment (LavanyaAnaimuthu) [reply README.md]: changed.

File: README.md (hunk 3 above)
Comment (LavanyaAnaimuthu) [reply README.md]: changed.

**Pull request merged**